client/common/network.go

```diff
@@ -122,6 +122,15 @@ func LoadNetwork(name string, node string) Network {
 			explorerGrpcEndpoint = "tcp://k8s.mainnet.explorer.grpc.injective.network:443"
 			exchangeTlsCert = credentials.NewServerTLSFromCert(&tls.Certificate{})
 			explorerTlsCert = LoadTlsCert(certPath, explorerGrpcEndpoint)
+		} else if node == "sentry" {
+			lcdEndpoint = "http://sentry.lcd.injective.network:10337"
```

Comment:
We could use HTTPS endpoints to have consistency for LCD & TM.

https://sentry.lcd.injective.network/
https://sentry.tm.injective.network/